apycula/tiled_fuzzer.py

```diff
@@ -107,6 +107,12 @@ def recode_idx_gw1n4(idx):
         "partnumber": "GW1N-LV1LQ144C6/I5",
         "recode_idx": recode_idx_gw1n1,
     },
+    "GW1NZ-1": {
+        "package": "QFN48",
+        "device": "GW1NZ-1-QFN48-6",
+        "partnumber": "GW1NZ-LV1QN48C6/I5",
+        "recode_idx": recode_idx_gw1n1, # TODO: check
```

Comment:
At a quick glance there are significant differences from GW1N-1, so it needs its own index recoding table. I can do it next week if you don't do it before then.

Reply:
Hm you think the recoding could affect the clock fuzzer? Probably not right?

Reply:
Clock fuzzer? Oh, I don't think so. I'm only looking at I/O. The clock fuzzer should not be affected in any way.

Reply:
Yea I looked at the tile grid and I think I know what the problem is with the clock fuzzer, pushed a fix, so we'll know soon enough.

Reply:
This is an example of a theoretical recoding based on the numbers in the tables. Requires experimental verification to be sure, of course. 
```
def recode_idx_gw1nz_1(idx):
    new_idx = idx
    if idx >= 40:
        new_idx -= 10
    if idx >= 60:
        new_idx -= 2
    if idx >= 80:
        new_idx -= 6
    return new_idx
```

Reply:
@chiplet are you able to test this? It'll be a while before I have a new Tang Nano 1K.

Reply:
Here is an example of adding a new chip on the nextpnr side: https://github.com/YosysHQ/nextpnr/pull/909/files

Reply:
At least the blinky seems to work now! Does this mean that the flow works correctly for all IO or should I manually verify each pin as well?